templates/archives/stories/time_records.jinja2

```diff
@@ -1,39 +1,44 @@
-    <img src="{{ static('img/stories/sample_story.jpg') }}"
-         class="class="figure-img img-fluid rounded" 
-         alt="A cat operating a computing device" />
-    <figcaption class="figure-caption">Cats love computers.</figcaption>
+    <img src="{{ static('img/stories/time_records-teaser.png') }}"
+         class="class=figure-img img-fluid rounded"
+         alt="Whirlwind 1" />
+    <figcaption class="figure-caption"></figcaption>
 </figure>
 
-<p>The types of problems that these laboratories tackled were far too long to do by hand. In
-    fact, even with a computer they still took many hours. One person by the name of Sam H.
-    Davis, Jr. had spend over 25 hours on a single problem. At an average rate of $250/hr, he
-    would have had to spend over $6,300 to solve this problem. If you thought this was expensive,
-    another person by the name of Jack L. Uretsky had spent over 73 hours to solve a problem,
-    costing him a grand total of nearly $20,000! This cost is without even considering the price
-    of each team member in the lab. Considering how expensive these problems were with the help of a computer, imagine how much
-    more expensive it would have been in terms of time to do it by hand. The Whirlwind 1 is able
-    to do 50,000 additions per second and these problems still took over 3 days of constant work
-    with an entire laboratory to solve. By hand, this may have taken months or years. Even
-    creating the computer was insanely expensive. 175 people along with a million dollar a year
-    budget and it still took over 3 years to build.
+<p>
+    The types of problems that these laboratories tackled were far too long to do by hand. In
+    fact, even with a computer they still took many hours. One person by the name of <a
+        href="/archives/doc/1_5_whirlwind_time_134">Sam H. Davis, Jr.</a> had spend over 25 hours on a single
+    problem. At an average rate of $250/hr, he would have had to spend over $6,300 to solve this
+    problem. If you thought this was expensive, another person by the name of <a
+        href="/archives/doc/1_5_whirlwind_time_77">Jack L. Uretsky</a> had spent over 73 hours to solve a problem,
+    costing him a grand total of nearly $20,000! Taking into account inflation, this comes out to
+    around $190,000 in 2019. This cost is without even considering the price of each team member
+    in the lab. Considering how expensive these problems were with the help of a computer,
+    imagine how much more expensive it would have been in terms of time to do it by hand. The
+    Whirlwind 1 is able to do 50,000 additions per second and these problems still took over 3
+    days of constant work with an entire laboratory to solve. By hand, this may have taken months
+    or years. Even creating the computer was insanely expensive. One hundred and seventy-five
+    people along with a million dollar a year budget and it still took over 3 years to build.
```

Comment:
What project is this that has 175 people, etc? If you name it we could have more context and detail to make your contribution more effective.